bin/.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
```

Comment:
There is a whole series of files included here under the bin directory; that can't be right.

Reply:
@dbmalkovsky do you mean the bin folder (just removed) or something related to this file maven.yml?

Reply:
@mwithi I meant the bin folder; this this conversation is resolved.